bot.py

```diff
@@ -207,22 +239,22 @@ async def uwu(ctx, arg1=""):
 
         if(not argIsText):
             message = await channel.fetch_message(msg_id)
-            if "uwu-immune" in [r.name.lower() for r in message.author.roles]:
+            if message.author.name + message.author.discriminator in immuwune:
                 await ctx.send("UwU this usew is uwu-immune! sowwy... 😭")
                 return
             else:
                 message = message.content
         else:
             # arg1 is original text that wants to be uwu-ized
-            if "uwu-immune" in [r.name.lower() for r in ctx.message.author.roles]:
+            if ctx.message.author.name + ctx.message.author.discriminator in immuwune:
                 await ctx.send("UwU you'we immune to the uwu! sowwy... 😭")
```

Comment:
Here as well

Reply:
🤔I mean....I guess. Especially with this new word I've coined.

Reply:
Your call, it doesn't affect the PR.

Reply:
Well I did it anyway so :P